client/src/pages/Login.js

```diff
@@ -1,7 +1,106 @@
-import { Box } from "@material-ui/core";
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core'
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import useStyles from '../styles'
+import { useForm } from 'react-hook-form'
+import green from '@material-ui/core/colors/green'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Checkbox from '@material-ui/core/Checkbox'
 
-function Login() {
-  return <Box>Login Page</Box>;
+
+const Login = () => {
+    const classes = useStyles()
+
+    const { register, handleSubmit, formState: { errors }, watch, setValue } = useForm({
+        defaultValues: { remember: false }
+    })
+
+    const onSubmit = (formData) => {
+        console.log(formData)
+    }
+
+    return (
+        <Grid container item direction='column' xs={12} spacing={6}>
```

Comment:
The inputs for this and the signup page are off center, you should be able to fix that with the justify/alignItems property

Reply:
yes, it is not centered. I will fix it.